src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/modifiableCircle.scss

```diff
@@ -0,0 +1,67 @@
+@import '../../containers/Application/colors.scss';
+
+$backgroundColor: rgba($black, .6);
+$borderColor: $white;
+$borderSize: 2px;
+
+.circle {
+    position: absolute;
+    background: $backgroundColor;
+    border-radius: 100%;
+    transform: translate(-50%, -50%);
+    will-change: width, height, left, top;
+    cursor: move;
+    border: $borderSize solid $borderColor;
+
+    &.disabled {
+        cursor: default;
+        border: none;
+        pointer-events: none;
+    }
+
+    &.filled {
+        background: $white;
+        box-shadow: 0 0 2px 1px rgba($black, .6);
+
+        &.disabled {
+            background: rgba($white, .6);
+        }
+    }
+}
+
+.label {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    font-size: 34px;
```

Comment:
yes, i guess this can be removed

Reply:
Then I would prefer to do so, because otherwise some developer in the future (maybe it is even you :wink:) might wonder why the text has a different font size.